lib/screens/edit_product_screen.dart

```diff
@@ -86,6 +98,49 @@ class _EditProductScreenState extends State<EditProductScreen> {
     );
   }
 
+  String validatorImageUrl(String value) {
+    if (value.isEmpty) {
+      return 'Please enter an image URL';
+    }
+    if (value.startsWith('http') && !value.startsWith('https')) {
+      return 'Please enter a valid URL.';
+    }
+    if (!value.endsWith('.png') && !value.endsWith('jpg') && !value.endsWith('.jpeg')) {
+      return 'Please enter a valid image URL.';
+    }
+    return null;
+  }
+
+  String validatorDescription(String value) {
+    if (value.isEmpty) {
+      return 'Please enter a description';
+    }
+    if (value.length < 10 != null) {
+      return 'Should be at least 10 characters long.';
+    }
+    return null;
```

Comment:
try:

`return value.isEmpty ? 'Please enter a description' : value.length < 10 != null ? 'Should be at least 10 characters long.' : null;`

Reply:
![image](https://user-images.githubusercontent.com/74462760/104463683-0b4ae580-55ed-11eb-8de8-30342a5db55e.png)

Even if the characters are more than 10, it still shows the validation.

Reply:
I see. Then, we will not change anything here